nucypher/__init__.py

```diff
@@ -1,9 +1,17 @@
 
-
 # Set Default Curve #
 #####################
 
 from umbral.config import set_default_curve
 set_default_curve()
 
 #####################
+
+
+# Report to Sentry #
+####################
+
+import sentry_sdk
```

Comment:
For this to work, need `sentry_sdk` in `Pipfile`. (Is it always necessary, can we be ok with ImportError?)

Reply:
Good Question - How option do we want this to be for now?

Reply:
What if we set up sentry_sdk when first setting up ursula configs, having using sentry as "yes" by default?

Reply:
Yeah - not a bad Idea.   Moving to #512